### ISSUE TITLE
Version selector appears even with 'version_selector': False

### ISSUE THREAD

**Issue by Hyrla** [readthedocs/sphinx_rtd_theme#1620]
### Problem

The version selector appears on the top left even if there is only one version and version_selector is set to False in html_theme_options.

### Reproducible Project

Project documentation: https://esieabot.readthedocs.io/en/latest/
Project documentation conf.py file: https://gitlab.esiea.fr/esieabot/esieabot-docs/rtd/-/blob/main/docs/source/conf.py

#### Error Logs/Results

![image](https://github.com/user-attachments/assets/c435ae3b-eb79-4f79-93d9-f295a3994b90)

#### Expected Results

It think it was expected that the version selector shouldn't be there at all?

### Environment Info

- Python Version: 3.12
- Sphinx Version: 8.3.1
- RTD Theme Version: latest (3.0.1)

**Comment by humitos** [comment]
Hrm, this is due this code: https://github.com/readthedocs/sphinx_rtd_theme/blob/7b894f5e7a5053dd35de406bc779020c229f20a9/sphinx_rtd_theme/static/js/versions.js_t#L2-L3

which renders as:

```
const themeVersionSelector = "False";
const themeLanguageSelector = "True";
```

We should update it to use a filter to make `True` -> `true` and remove the quotes.

**Comment by humitos** [comment]
I just released v3.0.2 that includes this fix. Please let us know if it works as expected.

**Comment by Hyrla** [comment]
> I just released v3.0.2 that includes this fix. Please let us know if it works as expected.

Yes issue has been fixed, thank you very much!
